### ISSUE TITLE
aria-controls does not work in JAWS 2018

### ISSUE THREAD

**Issue by gdkraus** [FreedomScientific/VFO-standards-support#70]
### Summary

JAWS 2018 announces that you can press JAWS Key + Alt + M for components that have aria-controls pointing to a valid target, but pressing the key combination does not do anything.

### Expected result

The virtual cursor should have moved to the controlled element.

### Actual result

Nothing happens in JAWS 2018 or JAWS 18, but it does work in JAWS 17

### Example

https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html

#### JAWS Version
 JAWS 2018.1803.24
 
#### Operating System and version
Windows 10 Enterprise
Version 1709
OS Build 16299.309

#### Browser and version:
IE 11
Chrome 65

**Comment by gdkraus** [comment]
We've done some additional research on this and found the source of the problem. It has to do with what keyboard layout you have JAWS set up to use. If you have JAWS set up for laptop layout, if you use CAPS LOCK + ALT + M, everything works. If you use INS + ALT + M, it does not work. However, even in laptop mode, the INS key still seems to function as a valid JAWS key for other JAWS functions. Using INS in laptop layout does not seem to work for following aria-controls.

If you are in desktop keyboard layout, using INS + ALT + M works for aria-controls.

**Comment by gdkraus** [comment]
One other issue to consider with this is what should the expected behavior be when the the virtual cursor should remain off, like for an application which is basically a large role="application". Does it make sense to announce that the user can move to a controlled element when the virtual cursor is off? Does moving to a controlled element always assume that it's a move of the virtual cursor, and could not be a keyboard focus move? Should the announcement of aria-controls be different if it is inside of a role="application"?
